net/coova-chilli/Config.in

```diff
@@ -32,7 +32,7 @@ config COOVACHILLI_JSONINTERFACE
 
 choice
 	prompt "SSL library"
-	default COOVACHILLI_NOSSL
+	default COOVACHILLI_WOLFSSL
```

Comment:
I havent go through entire PR, but is there any reason, why you choose wolfssl instead of openssl?
Variants having `coova-chilli-openssl` and `coova-chilli-wolfssl` sounds like better option here for end-users, though.

Reply:
Definitely it would be better to do variants. I got super busy with Google Summer of Code thanks to which we're also working on improving the [uspot captive portal](https://summerofcode.withgoogle.com/programs/2025/projects/292OF54a).

Due to this work on uspot, I will hardly find the time now to complete the changes here. If uspot works as we need in OpenWISP we'll likely switch to uspot for good. If not I will come back and complete this, but I don't want to leave lingering PRs here so I'll close this for now.

Thanks for your reviews guys!